src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationResource.kt

```diff
@@ -124,4 +126,107 @@ class ActivitiesMigrationResource(
       includeOnlyFailures = includeOnlyFailures,
     ),
   )
+
+  @PreAuthorize("hasRole('ROLE_MIGRATE_ACTIVITIES')")
+  @GetMapping("/activities/history/{migrationId}")
+  @Operation(
+    summary = "Gets a specific migration history record",
+    description = "Requires role <b>MIGRATE_ACTIVITIES</b>",
+    responses = [
+      ApiResponse(
+        responseCode = "200",
+        description = "The migration history record",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = MigrationHistory::class),
```

Comment:
odd to be specifying the content implementation here - thought it normally was filled in automatically anyway?